### DISCUSSION TITLE
The maximum iso value in Log mode.

### DISCUSSION THREAD

**Discussion by Inter9800** [gopro/labs#987] [General]
The question arose, is it possible in Log mode to set the maximum iso value above the standard 1600? I am experimenting with the LOGB parameter, I would like to see the result with an ISO value above the standard.

**Comment by Deadcode85** [comment]
You can simply change to ISO Max to 3200 or 6400 in the gopro menu. However according to my test, you won't get any benefits by rising the ISO, the results are the same if boosted from ISO1600 (or ISO800...) source... but with lower ISO you will have more highlight headroom, same noise.

**Reply by Inter9800** [reply]
In Log mode, the maximum ISO value is 1600, I can't use standard tools to select a higher value, but I would like to.

**Comment by Deadcode85** [comment]
Switch to regular GoPro Flat, and show this command to the camera:
![IMG_3090](https://github.com/user-attachments/assets/f374aa2e-aacf-4c5d-a999-9278de29ff56)

With this you have same image as in LOG, but with more controls.
